docs/source/upcoming_release_notes/1068-changing_sp1k4_two_new_motors_name.rst

```diff
@@ -0,0 +1,31 @@
+1068 changing sp1k4 two new motors name
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- N/A
+
+Device Updates
+--------------
+lens_pitch to lens_pitch_up_down
+lens_yaw to lens_yaw_left_right
+
+New Devices
+-----------
+- N/A
+
+Bugfixes
+--------
+- N/A
+
+Maintenance
+-----------
+- N/A
+
+Contributors
+------------
+tongju@slac.stanford.edu
```

Comment:
```suggestion
- tongju12
```

Reply:
I accidentally close this pull request and reopen again. Sorry about it. Can you approve again? @tangkong